wordpress-mobile/WordPress-Android
# Feature/prologue refresh

This PR introduces a new login prologue.

It might seem really big, but most of it just layout changes, which you can confirm just by running an app.

Portrait:
[![Image from Gyazo](https://i.gyazo.com/df09dcfec998cb3a17046e21c8c3b184.jpg)](https://gyazo.com/df09dcfec998cb3a17046e21c8c3b184)

Landscape:
[![Image from Gyazo](https://i.gyazo.com/67af944cc399c6d79abf72c15ccc5285.png)](https://gyazo.com/67af944cc399c6d79abf72c15ccc5285)

Tablet:
[![Image from Gyazo](https://i.gyazo.com/763cc9cb847c81aec5d5341276c70082.jpg)](https://gyazo.com/763cc9cb847c81aec5d5341276c70082)

To make the content of each slide look right on each screen size we calculate a scaling factor and scale the layout accordingly.

The transparent status bar is supported starting from API 23.

To test:
- Clear the app data and install the app from this PR.
- Check that the login prologue is visually solid.
- Navigate into login flow and make sure the status bar looks ok in both Light and Dark modes and in Landscape/Portrait orientations.
- Perform the same checks on a device with API 21.

PR submission checklist:
